src/UNL/Officefinder.php

```diff
@@ -115,22 +114,7 @@ public function __construct($options = [])
     protected static function _getAuth()
     {
         if (!self::$auth) {
-            $protocol = new SimpleCAS_Protocol_Version2([
-                'hostname' => 'login.unl.edu',
-                'port' => 443,
-                'uri' => 'cas',
-                'request' => new HTTP_Request2(null, null, [
-                    'adapter' => 'HTTP_Request2_Adapter_Curl'
-                ]),
-            ]);
-            $cacheDriver = new \Stash\Driver\FileSystem();
-            $cacheDriver->setOptions([
-                'path' => realpath(__DIR__ . '/../..') . '/tmp/simpleCAS_map',
-            ]);
-            $sessionMap = new SimpleCAS_SLOMap($cacheDriver);
-            $protocol->setSessionMap($sessionMap);
-
-            self::$auth = SimpleCAS::client($protocol);
+            self::$auth = new UNL_Officefinder_Auth();
```

Comment:
Very similar to what you are doing, but 6.1 has an `on_stats` callback with the effective url. http://docs.guzzlephp.org/en/latest/request-options.html#on-stats